lib/util.js

```diff
@@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (Number.isNaN(value.getTime())) {
+        return ctx.stylize(value.toString(), 'date');
+      } else {
+        return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      }
```

Comment:
just a style nit so feel free to ignore...

```
return ctx.stylize(Number.isNaN(value.getTime()) ?
                     value.toString() :
                     Date.prototype.toISOString.call(value),
                   'date');
```

... would help eliminate some minor code duplication